app/models/data_resources/generic_item.rb

```diff
@@ -0,0 +1,67 @@
+class GenericItem < ApplicationRecord
+  include FilterByRole
+  has_ancestry orphan_strategy: :destroy
+
+  validates_presence_of :generic_type
+  store :payload, coder: JSON
+
+  belongs_to :data_provider
+  has_one :external_reference, as: :external, dependent: :destroy
+  has_many :accessibility_informations, as: :accessable, dependent: :destroy
+  has_many :addresses, as: :addressable, dependent: :destroy
+  has_many :data_resource_categories, as: :data_resource
+  has_many :categories, through: :data_resource_categories
+  has_many :contacts, as: :contactable, dependent: :destroy
+  has_many :companies, as: :companyable, class_name: "OperatingCompany", dependent: :destroy
+  has_many :content_blocks, as: :content_blockable, dependent: :destroy
+  has_many :web_urls, as: :web_urlable, dependent: :destroy
+  has_many :opening_hours, as: :openingable, dependent: :destroy
+  has_many :price_informations, as: :priceable, class_name: "Price", dependent: :destroy
+  has_many :media_contents, as: :mediaable, dependent: :destroy
+  has_many :locations, as: :locateable, dependent: :destroy
+
+  scope :with_category, lambda { |category_id|
+    where(categories: { id: category_id }).joins(:categories)
+  }
+
+  accepts_nested_attributes_for :web_urls, reject_if: ->(attr) { attr[:url].blank? }
+  accepts_nested_attributes_for :content_blocks, :data_provider, :price_informations, :opening_hours,
+                                :media_contents, :accessibility_informations, :addresses, :contacts,
+                                :companies, :locations
```

Comment:
`acts_as_taggable`?  gibt es sonst auch bei allen haupt-datentypen

Reply:
zusammen mit `argument :tags, [String], as: :tag_list, required: false`
Ist das beim GenericItem Interessant, dann baue ich es mit ein?

Reply:
keine ahnung, haben alle. ist es denn bei den andere interessant überhaupt?

Reply:
ich habe es noch nie verwendet, in der App bisher genutzt?

Reply:
nein